invenio.cfg

```diff
@@ -333,6 +334,8 @@ CDS_EOS_OFFLOAD_REDIRECT_BASE_PATH = ""
 CDS_CERN_SCIENTIFIC_COMMUNITY_ID = "CHANGE_ME"
 """The id of the CERN Scientific community."""
 
+CDS_CERN_MINT_ALTERNATE_IDS = {"cdsrn": "CDS Reference"}
```

Comment:
didn't we say to have `CDS report number` as a name in the end? @ntarocco

Reply:
I would say so, what was the outcome to this question in your interviews? (pinging @zzacharo too)